admin/sql/create_tables.sql

```diff
@@ -5,25 +5,45 @@ CREATE TABLE lowlevel (
   mbid        UUID NOT NULL,
   build_sha1  TEXT NOT NULL,
   lossless    BOOLEAN                  DEFAULT 'n',
-  data        JSON NOT NULL,
-  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
-  data_sha256 TEXT NOT NULL
+  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+);
+
+CREATE TABLE lowlevel_json (
+  id          INTEGER, -- FK to lowlevel.id
+  data        JSONB    NOT NULL,
+  data_sha256 CHAR(64) NOT NULL
 );
 
 CREATE TABLE highlevel (
   id         INTEGER, -- FK to lowlevel.id
   mbid       UUID    NOT NULL,
   build_sha1 TEXT    NOT NULL,
-  data       INTEGER NOT NULL, -- FK to highlevel_json.data
   submitted  TIMESTAMP WITH TIME ZONE DEFAULT NOW()
 );
 
-CREATE TABLE highlevel_json (
-  id          SERIAL,
-  data        JSON     NOT NULL,
+CREATE TABLE highlevel_meta (
+  id          INTEGER, -- FK to highlevel.id
+  data        JSONB    NOT NULL,
   data_sha256 CHAR(64) NOT NULL
 );
 
+CREATE TABLE highlevel_model (
+  id          SERIAL,
+  highlevel   INTEGER, -- FK to highlevel.id
+  data        JSONB    NOT NULL,
+  data_sha256 CHAR(64) NOT NULL,
+  model       INTEGER -- FK to model.id
+);
+
+CREATE TABLE model (
+  id            SERIAL,
+  model         TEXT NOT NULL,
+  model_version TEXT NOT NULL,
+  build_sha1    TEXT NOT NULL,
```

Comment:
In the highlevel json file (http://acousticbrainz.org/98255a8c-017a-4bc7-8dd6-1fa36124572b/high-level?n=0) we have git sha, version, and build sha for both essentia and gaia:
```
version: {
  highlevel: {
    essentia: "2.1-beta1",
    essentia_build_sha: "8e24b98b71ad84f3024c7541412f02124a26d327",
    essentia_git_sha: "v2.1_beta1-228-g260734a",
    extractor: "music 1.0",
    gaia: "2.4-dev",
    gaia_git_sha: "857329b",
    models_essentia_git_sha: "v2.1_beta1"
  },
  lowlevel: {
    essentia: "2.1-beta2",
    essentia_build_sha: "c0e182ba78c2de26e491b7b7a4a2c98c1ed34208",
    essentia_git_sha: "v2.1_beta2-2-gbb40004",
    extractor: "music 1.0"
  }
}
}
```
I don't know what data we should store in this table. Git revisions seem more useful that sha1 of the binary. Should we have essentia or gaia?